cmd/main.go

```diff
@@ -96,10 +96,12 @@ func main() {
 	log.Info("starting NSX Operator")
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		HealthProbeBindAddress: probeAddr,
-		MetricsBindAddress:     metricsAddr,
-		LeaderElectionID:       "nsx-operator",
+		Scheme:                  scheme,
+		HealthProbeBindAddress:  probeAddr,
+		MetricsBindAddress:      metricsAddr,
+		LeaderElection:          true,
+		LeaderElectionNamespace: "default",
```

Comment:
What is the election namespace used for? Will it create any resource in the namespace

Reply:
Yes, it would create `lease` lock in the namespace, currently put it in default namespace.

Reply:
Could we put it in nsx-operator namespace?

Reply:
The namespace of WCP and vanilla k8s is different, so I put in the default.

Reply:
yes, in WCP, the ncp is in vmware-system-nsx and in k8s, its in nsx-system
but, we can also change k8s env setup and put ncp and nsx-oprator in vmware-system-nsx as well. 
In such case, we just use ncp from photon image.

Reply:
OK, read it from env